game/src/main/scala/io/aigar/controller/GameResponse.scala

```diff
@@ -0,0 +1,48 @@
+package io.aigar.controller.response
+
+case class Position(
+  x: Float,
+  y: Float
+)
+case class Direction(
+  x: Float,
+  y: Float
+)
```

Comment:
I'm underqualified for this conversation, but if our direction is not normalized, would it mean that we already have the velocity?